manifests/base/webhook/service.yaml

```diff
@@ -8,12 +7,16 @@ metadata:
     prometheus.io/port: "8080"
   labels:
     app: training-operator
-  name: training-operator
+  name: training-operator-service
```

Comment:
@tenzen-y Do you need to move this service to the webhook  and name it as `training-operator-service`?
Why we can't just keep service name as it is in `/manifests/base/service.yaml` and just add another 9443 port ?

Reply:
Like what we do in Katib: https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/components/controller/service.yaml

Reply:
Done.